src/halfpipe/tui/data_analyzers/meta_data_steps.py

```diff
@@ -54,6 +54,8 @@ def _get_field(schema, key):
         instance = schema()
     else:
         instance = schema
+    print("iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii", instance, schema, key)
```

Comment:
Could you try using logging statements instead?

Reply:
sorry, this was there for debugging purposes and i forgot to delete it